testdata/p4_16_samples_outputs/issue513-midend.p4

```diff
@@ -0,0 +1,68 @@
+#include <core.p4>
+#define V1MODEL_VERSION 20180101
+#include <v1model.p4>
+
+header Ethernet_h {
+    bit<48> dstAddr;
+    bit<48> srcAddr;
+    bit<16> etherType;
+}
+
+struct Parsed_packet {
+    Ethernet_h ethernet;
+}
+
+struct mystruct1 {
+    bit<4> a;
+    bit<4> b;
+}
+
+control DeparserI(packet_out packet, in Parsed_packet hdr) {
+    apply {
+        packet.emit<Ethernet_h>(hdr.ethernet);
+    }
+}
+
+parser parserI(packet_in pkt, out Parsed_packet hdr, inout mystruct1 meta, inout standard_metadata_t stdmeta) {
+    state start {
+        pkt.extract<Ethernet_h>(hdr.ethernet);
+        transition accept;
+    }
+}
+
+control cIngress(inout Parsed_packet hdr, inout mystruct1 meta, inout standard_metadata_t stdmeta) {
+    @name("cIngress.foo") action foo() {
+        meta.b = meta.b + 4w5;
+        mark_to_drop();
```

Comment:
@mbudiu-vmw Should we use `mark_to_drop(stdmeta)` here to avoid the warning about deprecated feature?

Reply:
I just moved this test from err, without any other changes.